app/assets/stylesheets/pageflow/navigation_mobile.css.scss

```diff
@@ -99,6 +99,44 @@ body {
     }
   }
 
+  .parent_page {
+    @include background-icon-center($color: #ddd);
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.